pools/resource_instance_pool.go

```diff
@@ -29,7 +29,8 @@ type InstancePoolAzureAttributes struct {
 // https://docs.gcp.databricks.com/dev-tools/api/latest/instance-pools.html#instancepoolgcpattributes
 type InstancePoolGcpAttributes struct {
 	Availability  clusters.Availability `json:"gcp_availability,omitempty" tf:"force_new"`
```

Comment:
should this also be `tf:"computed,force_new"`?

Reply:
It has default, so it's not `computed`: https://github.com/databricks/terraform-provider-databricks/blob/main/pools/resource_instance_pool.go#L200